lib/Doctrine/ORM/Cache/DefaultCacheFactory.php

```diff
@@ -217,7 +217,7 @@ public function getRegion(array $cache)
 
             if ( ! $this->fileLockRegionDirectory) {
                 throw new \LogicException(
-                    'If you what to use a "READ_WRITE" cache an implementation of "Doctrine\ORM\Cache\ConcurrentRegion" is required, ' .
+                    'If you want to use a "READ_WRITE" cache an implementation of "Doctrine\ORM\Cache\ConcurrentRegion" is required, ' .
                     'The default implementation provided by doctrine is "Doctrine\ORM\Cache\Region\FileLockRegion" if you what to use it please provide a valid directory, DefaultCacheFactory#setFileLockRegionDirectory(). '
```

Comment:
Same typo next line (starting with `The default implementation provided by doctrine is...`)